docs/development.md

```diff
@@ -14,6 +14,12 @@
 
 ## Setting up development environment
 
+- Prerequisite for development environment
```

Comment:
You could possibly reword this to `Install yarn` to be more consistent with the other items on the page, which call out what the step actually is (`Install dependencies`, `Install node` etc). Not a blocker though.